deploy/kube/conf/import_dashboard.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+AddDataSource() {
+  curl 'http://localhost:3000/api/datasources' \
+    -X POST \
+    -uadmin:admin\
+    -H 'Content-Type: application/json;charset=UTF-8' \
+    --data-binary \
+    '{"name":"Prometheus","type":"prometheus","url":"http://prometheus:9090","access":"proxy","isDefault":true}'
+}
+
+ImportDashboard() {
+  if [[ ! -z ${DASHBOARD_URL} ]];then
+    curl ${DASHBOARD_URL} -o  /grafana-dashboard.json
+  fi
+  export FL="/tmp/dash"
+
+  echo "{ \"overwrite\": true, \"dashboard\": " > $FL
+  sed 's/${DS_PROMETHEUS}/Prometheus/g' /grafana-dashboard.json >> $FL
```

Comment:
i think we should comment on the why here. Anyone not in the room as we were debugging this will be mystified by this when they stumble upon it years from now.